recipes/superlu_dist/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "5.2.2" %}
+
+{% set mpi = 'mpich' %}
+
+package:
+  name: superlu_dist
+  version: {{ version }}
+
+source:
+  fn: superlu-{{ version }}.tar.gz
+  url: http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_dist_{{ version }}.tar.gz
+  sha256: bd13e694bae1d703e66d8efd8f67e31fb0b68466b248a37ed165b371945d49ca
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+  host:
+    - cmake
+    - {{ mpi }}
+    - metis
+    - openblas
+    - parmetis
+  run:
+    - {{ mpi }}
+    - metis
+    - openblas
+    - parmetis
+    - llvm-openmp  # [osx]
```

Comment:
- should we require `blas * openblas` instead of openblas directly?
- or should we require `blas` and build with different blas variants?

- do metis and parmetis have run exports that may allow us to not make them run dependencies.

Reply:
cc @minrk